01/assets/js/app.js

```diff
@@ -1,4 +1,22 @@
-const a = '4.2';
-const b = 9;
+let a = parseInt('4.2');
+let b = 9;
 
-console.log(a, b);
\ No newline at end of file
+console.log(a, b);
+console.log(typeof a, typeof b);
+
+const addition = a + b;
+const subtraction = a - b;
+const multiplication = a * b;
+const exponentiation = a ** b;
+const division = a / b;
+const modulus = a % b;
+
+const checkNumber = (operation, operationName = 'operation') => {
+	console.log(`Result: ${operation}`);
+	if (operation >= 20) {
+		console.log(`Result of ${operationName} is greater or equal 20`);
+	} else {
+		console.log(`Result of ${operationName} is less than 20`);
+	}
+};
+checkNumber(subtraction, 'addition');
```

Comment:
Niestety wykorzystanie budzi pewne wątpliwości czy jest to poprawnie zrobione.
Zobacz, pierwszy parametr ma nazwę "subtraction", drugi "addition" - widząc to od razu myślę, że coś tu nie gra :D
Może rozwiązanie nasunie Ci ten artykuł: https://devmentor.pl/b/mniej-instrukcji-warunkowych

Reply:
Nie wiem czy do końca chodziło o takie rozwiązanie, chyba na nic innego aktualnie nie jestem w stanie wpaść 😛

Reply:
Skomentowane w kodzie ;)